src/sincosint.jl

```diff
@@ -18,7 +18,7 @@ using Base.Math.@horner
 function sinint(x::Float64)
     t = x*x
     if t ≤ 36.0
-        return x * @horner(t, 1.00000000000000000000E0,
```

Comment:
I was just trying to understand why the coverage tests flagged these lines as being untested. In the end it had no effect so I agree there is nothing gained by the change. I just forgot to undo the change in my branch, sorry.